dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -193,7 +193,7 @@ public void ValidateGetSetBackground()
             });
         }
 
-        [TestMethod]
+        //[TestMethod] disabled with issue #1815
```

Comment:
This was unreliable only in specific configurations, so perhaps we could still have it running in others ?

Reply:
Yeah, we can probably just revert this https://github.com/microsoft/microsoft-ui-xaml/pull/1797, which will skip debug builds.